packages/rocketchat-lib/server/functions/setRealName.coffee

```diff
@@ -0,0 +1,24 @@
+RocketChat._setRealName = (userId, name) ->
```

Comment:
It's how `setUsername` is done, so I just copied that. I think its because it has to make more requests to other models, so it's not self contained within one model.